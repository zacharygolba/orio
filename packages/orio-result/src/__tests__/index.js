// @flow

import * as result from '../'

test('#done()', () => {
  expect(result.done()).toMatchSnapshot()
  expect(result.done('test')).toMatchSnapshot()
})

test('#next()', () => {
  expect(result.next('test')).toMatchSnapshot()
})
