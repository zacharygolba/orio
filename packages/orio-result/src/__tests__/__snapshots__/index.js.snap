// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#done() 1`] = `
Object {
  "done": true,
  "value": undefined,
}
`;

exports[`#done() 2`] = `
Object {
  "done": true,
  "value": "test",
}
`;

exports[`#next() 1`] = `
Object {
  "done": false,
  "value": "test",
}
`;
